### PR TITLE
Fix Statemine ss58 prefixes.

### PR DESCRIPTION
**Pull request by riusricardo**

### CODE REVIEW

**Review by hbulgarini — APPROVED** [review]
LGTM! i wonder if we should do the same for statemint on the file: https://github.com/paritytech/cumulus/blob/master/polkadot-parachains/res/statemint.json

**Comment by riusricardo** [comment]
@hbulgarini I did the changes but the latest changes are on a different branch. The file you pointed out in master is from a test we did on Rococo.

**Comment by hbulgarini** [comment]
> @hbulgarini I did the changes but the latest changes are on a different branch. The file you pointed out in master is from a test we did on Rococo.

Ok! makes sense. Maybe this change should be included in the PR pointing to the release: https://github.com/paritytech/cumulus/pull/830

**Review by bkchr — APPROVED** [review]

File: polkadot-parachains/res/statemint.json [hunk 1]
```diff
@@ -9,7 +9,11 @@
              "/ip4/35.204.76.198/tcp/30333/p2p/12D3KooWAEeojfMRiSSZWDZBXKj7E6hx5p6J4KowMHb6C3uBEiwD"    ],
   "telemetryEndpoints": null,
   "protocolId": null,
-  "properties": null,
+  "properties": {
+    "ss58Format": 0,
+    "tokenDecimals": 10,
+    "tokenSymbol": "DOT"
+  },
```
[the remark this answers is not in the document]
Comment (riusricardo) [reply polkadot-parachains/res/statemint.json:16]: This is what I've been trying to say. This is why I made #830

[the remark this answers is not in the document]
Comment (riusricardo) [reply polkadot-parachains/res/statemint.json:16]: I'll revert this last commit here and reopen #830.

**Pull request merged**